core/src/main/java/com/google/instrumentation/stats/StatsManager.java

```diff
@@ -28,4 +28,9 @@
    * Returns the current stats data, {@link View}, associated with the given {@link ViewDescriptor}.
    */
   public abstract View getView(ViewDescriptor viewDescriptor);
+
+  /**
+   * Returns the default {@link StatsContextFactory}.
+   */
+  public abstract StatsContextFactory getStatsContextFactory();
```

Comment:
Please do not make this public until we decide how the contextFactory interface should look like and if it should be in the core as we have it in the trace. This means do not deprecate yet the old function and make this method protected.